src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py

```diff
@@ -16,7 +16,7 @@ class AIMLEmbeddingsComponent(LCEmbeddingsModel):
         DropdownInput(
             name="model_name",
             display_name="Model Name",
-            options=AIML_EMBEDDING_MODELS,
+            options=AimlModels().embedding_models,
```

Comment:
```suggestion
            options=[]
```
This should be delayed because otherwise it will load models that might never be used for certain users.

Reply:
I also wouldn't expect this to work, in general. Doesn't the class need to call `get_aiml_models` to populate the class variables?

Reply:
Hi! Thank you for your feedback and help, I really appreciate it.

I agree that your solution is better for now, so I’ve committed your changes.

To answer Jordan’s question, I think the class does need to call get_aiml_models to populate the class variables. However, I haven't rewritten the entire embeddings class at this time, as I don’t have the availability to do so right now. I do have a few ideas I’d like to integrate into the project in the future, so I’ll plan to revisit and improve this aspect a bit later.